examples/cluster_upgrade.yml

```diff
@@ -19,6 +19,10 @@
     host_names:
       - myhost1
       - myhost2
+    volumes:
+      - volume1
+      - volume2
+    check_upgrade: True
```

Comment:
is this required?

Reply:
It is example file. i added just for reference purpose.

Reply:
I think the example file should be as simple as possible and generic. This should go to README.md.

Reply:
Also, add the docs to README.md what the list of volumes mean and some information about it.